internal/store/postgres/alerts.go

```diff
@@ -99,7 +108,13 @@ func (r AlertRepository) List(ctx context.Context, flt alert.Filter) ([]alert.Al
 		return nil, err
 	}
 
-	rows, err := r.client.QueryxContext(ctx, query, args...)
+	// Instrumentation attributes
+	attrs := []attribute.KeyValue{
+		attribute.String("db.repository.method", "list"),
+		attribute.String("db.sql.table", "alerts"),
+	}
+
+	rows, err := r.client.QueryxContext(otelsql.WithCustomAttributes(ctx, attrs...), query, args...)
```

Comment:
Lets do something like this instead
```suggestion
        ctx = otelsql.WithCustomAttributes(ctx, []attribute.KeyValue{
		attribute.String("db.repository.method", "list"),
		attribute.String("db.sql.table", "alerts"),
	}...)
	
	rows, err := r.client.QueryxContext(ctx, query, args...)
```

Reply:
Done